bench.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ocaml/opam:debian-ocaml-5.3
+WORKDIR /bench-dir
+RUN sudo apt-get update && \
+    sudo apt-get install -qq -yy --no-install-recommends pkg-config libgmp-dev \
+      wget
+RUN sudo ln -sf /usr/bin/opam-2.3 /usr/bin/opam
+RUN opam remote add origin https://github.com/ocaml/opam-repository.git && \
+    opam update
+RUN wget https://nodejs.org/dist/v22.14.0/node-v22.14.0-linux-x64.tar.xz && \
+    tar xJvf node-v22.14.0-linux-x64.tar.xz && \
+    sudo cp node-v22.14.0-linux-x64/bin/node /usr/bin/node
+COPY --chown=opam:opam ./*.opam ./VERSION ./
+RUN opam exec -- git init && \
+    opam exec -- git add . && \
+    opam exec -- git commit -m dummy && \
+    opam exec -- git tag -a $(cat VERSION)-dev -m dummy
+RUN opam pin -yn --with-version=dev .
+RUN opam install -y --deps-only js_of_ocaml-compiler && \
+    opam install -y binaryen-bin
```

Comment:
```suggestion
```
I don't think this is useful, we're already installing wasm_of_ocaml-compiler below

Reply:
Yes, but having the installation of binaryen-bin in a different RUN command allows it to be cached separately. As a consequence, when wasm_of_ocaml-compiler is modified, the long build of binaryen will not be run again.

Reply:
Should we use `RUN opam install -y --deps-only wasm_of_ocaml-compiler` then ?

Reply:
Also, what do you think about using a released binary https://github.com/WebAssembly/binaryen/releases/tag/version_122 instead of recompiling it ?

Reply:
> Should we use `RUN opam install -y --deps-only wasm_of_ocaml-compiler` then ?

That would install js_of_ocaml-compiler, which requires the files to be copied first, which I wanted to avoid for the cache.

Maybe I should add comments to document.



> Also, what do you think about using a released binary https://github.com/WebAssembly/binaryen/releases/tag/version_122 instead of recompiling it ?

I thought about it, but it’s been a bit difficult due to the choice of installing binaryen as an opam package (binaryen-bin). I think we may want to come back on that choice.